app/controllers/people_controller.rb

```diff
@@ -14,10 +14,12 @@ def signin
   end
 
   def orgchart
-    @department = Department.where(shortname: "BAUX").first
-    @people = @department.people.active if @department
-    @page_title = "Org Chart"
+    @department = Department.find(params[:dept_id])
```

Comment:
This line causes issues for Codacy. I wonder if that's trying to tell us this is really about the department, not about people ?

Reply:
It seems to be complaining that I'm not scoping it correctly so that cancancan's ability.rb is enforced: https://github.com/CanCanCommunity/cancancan/wiki/authorizing-controller-actions

i can try `Department.accessible_by(current_ability).find(params[:dept_id])` or something to that effect.

Reply:
I was thinking the problem would go away if this were in the Departments controller. It's a report about a department, it should probably be there.